README.md

```diff
@@ -104,9 +102,11 @@ $ kubectl delete -f deploy/service_account.yaml
 $ kubectl delete -f deploy/crds/app_v1alpha1_appservice_crd.yaml
 ```
 
-## User Guide
+## User Guides
+
+To learn more about the writing an operator in Go, see the [user guide][guide].
 
-To learn more about the operator-sdk, see the [user guide][guide].
+The SDK also supports developing an operator using Ansible. See the [Ansible operator user guide][ansible_user_guide].
```

Comment:
I would add a note about this in the Workflow section as well. Perhaps @shawn-hurley or another Ansible-focused team member could write a workflow in this or a separate PR.

Reply:
I think we will add a follow up PR just to keep it easy